### DISCUSSION TITLE
The angular distributions of Bremsstrahlung radiation

### DISCUSSION THREAD

**Discussion by linlaakk** [nrc-cnrc/EGSnrc#1207] [Physics] [answered]
Hi,

Regarding MV linac simulations, where the electron beam collides with the high-density material X-ray target and interactions via bremsstrahlung radiation produce photons. How accurate would the angular distributions of Bremsstrahlung radiation model be?
The manual says that the implementation was adopted with slight modifications on page 71 "2.4.2.iii Simulation of discrete bremsstrahlung events, angular distribution". 
Would you have an opinion of which Geant4 physics list would be comparable to EGSnrc results?

Thanks and kind regards,
Linda

**Comment by ftessier** [comment]
This is an unresolved question. I hope @mainegra can chime in, as he knows most about this. Apparently, there is evidence that the brems angular distribution is not accurate enough in some cases. I don't know Geant4 enough to advise, maybe you can ask on the Geant4 repository, also on GitHub. I believe the Penelope manual can shed some light on this, if memory serves me well.

If you have gathered some additional literature on the subject, please feel free to post a list of references below!

**Reply by linlaakk** [reply]
Thanks for quick response! 
With an extremely simple simulation, 6 MeV electron beam bombarded to the high-density X-ray target, we have noticed a difference between EGSnrc and Geant4 Option4 physics list (recommended) in the angular distributions of Bremsstrahlung radiation model. The particle and momentum direction distributions are more peaked on the central axis with EGSnrc and the differences propagate into the dose distributions which causes problems when comparing the simulated data to the measurements.

We have found supporting literature for EGSnrc and Penelope:
- Pandola et al. (2015) reported that Penelope agreed better with experimental data in comparison to the previous standard package “Option3” and Livermore physics list
- Faddegon et al. (2008): EGSnrc and Penelope results closer to measurements, though, all three EGSnrc, Geant4 and Penelope codes overestimated the photon spectra at low energies

However, Penelope-2018 manual reports improvements on the intrinsic angular distribution of bremsstrahlung photons, and the results has been shown to lead to a better description of experimental spectra in the case of high-density targets (García-Higueras et at., 2023, https://doi.org/10.1016/j.radphyschem.2023.110949).

Geant4 physics manual says: 
- Option4 physics list (recommended) use similar bremsstrahlung photon angular sampling as EGSnrc without modifications: 2BS adapted from "Improved bremsstrahlung photon angular sampling in the EGS4 code system" by Alex F. Bielajew, Rahde Mohan and Chen-Shou Chui
- Option4 validated as recommended physics list in Arce et at. (2020) in Report on G4-Med, a Geant4 benchmarking system for medical physics applications developed by the Geant4 Medical Simulation Benchmarking Group
- Other physics list options available including Penelope physics list (based on Penelope-2008 code)

**Reply by ojalaj** [reply]
Hello EGSperts - by any chance, have you @ftessier @mainegra had a possibility to look at this further, using extra information provided by @linlaakk?

**Comment by mainegra** [comment, marked as answer]
Dear @linlaakk and @ojalaj 

The Working Group on Model-Based Dose Calculation Algorithms (WGMBDCA) in Brachytherapy has recently found about 15% differences in central axis depth dose in a water phantom for electronic brachytherapy sources that use transmission-mode Au foils (1 micron), while these differences are much less for reflexion-type thick W targets. A paper has been accepted for publication in Med Phys and as soon as it is available I can send you the link!
EGSnrc and Penelope use shape functions for the double differential bremsstrahlung cross section based on different models. EGSnrc relies on the work by [Koch and Motz 1959](https://doi.org/10.1103/RevModPhys.31.920) who used plane wave first Born approximations to obtain their formulas with and without screening corrections (KM model). Penelope as of their 2018 version moved to using the approach used by [Kissel, Quarles and Pratt 1983](https://doi.org/10.1016/0092-640X(83)90001-3) (KQP model) based on a relativistc partial wave approximation. The later should be considered theoretically more accurate within the grid used for energy and angle interpolations.

## Answer to @linlaakk question:

_Would you have an opinion of which Geant4 physics list would be comparable to EGSnrc results?_

As part of the work done by the WGMBDCA I learned that TOPAS, a GEANT4-based application for modelling LINACS can reproduce Penelope's and EGSnrc's results by using either `p4em-penelope` or `g4em-livermore` physics lists. The latter is not quite the same as what is done in EGSnrc as it simply uses the 2BN formula from the Koch and Motz paper. But probably good enough.

## Personal opinion

Before undertaking the labor-intensive task of generating shape functions for all Z materials across a reasonable 
(E,θ) interpolation grid, it is essential to gather additional experimental evidence to further support KQP. The currently available experimental data is hindered by significant uncertainties, making it challenging to draw definitive conclusions about which model provides a better fit to the experimental results.
